newIDE/app/src/EventsFunctionsExtensionEditor/index.js

```diff
@@ -56,6 +57,7 @@ type Props = {|
   onBehaviorEdited?: () => void,
   initiallyFocusedFunctionName: ?string,
   initiallyFocusedBehaviorName: ?string,
+  unsavedChangesManagement: UnsavedChanges,
```

Comment:
Nitpicking: We should make the variable name consistent with the type. Either
`unsavedChangesManagement: UnsavedChangesManagement,`
or
`unsavedChanges: UnsavedChanges,`

with a slight preference for `UnsavedChanges` as a usually read a type name as "this object represents TYPE" (This object represents UnsavedChanges).